packages/db/index.js

```diff
@@ -295,26 +316,26 @@ export class DBClient {
   /**
    * Check the email history for a specified email type to see if it has
    * been sent within a specified number of days. If not, it is resent.
-   * @param {import('./db-client-types').EmailSentRecentlyInput} email
+   * @param {import('./db-client-types').EmailSentInput} email
    * @returns {Promise<boolean>}
    */
-  async emailSentRecently (email) {
+  async emailHasBeenSent (email) {
     const {
       userId,
       emailType,
-      numberOfDays = 7
+      secondsSinceLastSent = 60 * 60 * 24 * 7
     } = email
 
-    const d = new Date()
-    d.setDate(d.getDate() - numberOfDays)
-    const numberOfDaysAgo = d.toISOString()
+    const now = new Date()
+    now.setSeconds(now.getSeconds() - secondsSinceLastSent)
```

Comment:
nit.

Given your setting this to be back in the past, calling it `now` is a bit confusing.

Reply:
True. Have renamed.

Reply:
Ta!